### PR TITLE
fix: fixed sc not working with siwe

### PR DESCRIPTION
**Pull request by JonathanConn**
Decided to go with infura
Will need an api key in our env vars for this
Tested with argent

### CODE REVIEW

File: src/handlers/verify.ts [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-      res
-        .status(422)
-        .json({ message: "Expected prepareMessage object as body." });
+      res.status(422).json({ message: 'Expected prepareMessage object as body.' });
       return;
     }
 
     const message = new SiweMessage(req.body.message);
-    const fields = await message.validate(req.body.signature);
+    const provider = new ethers.providers.InfuraProvider('mainnet', process.env.INFURA_API_KEY);
```
Comment (Cali93) [inline src/handlers/verify.ts:13]: Can we create the initiate the provider outside of the function ?

**Review by Cali93 — APPROVED** [review]
LGTM, just a minor tweak regarding the provider.

**Comment by JonathanConn** [comment]
Waiting on infura key
Needs to be added to gh env before merging or siwe will be broken

**Review by Cali93 — APPROVED** [review]
LGTM

**Pull request merged**